tests/phpunit/WC_Stripe_Payment_Gateway_Test.php

```
@@ -844,4 +845,148 @@ public function test_invalid_request_error_with_non_blocked_code_is_retryable()
+
+		add_filter( 'pre_http_request', $mock_request, 10, 3 );
+
+		$actual = $this->gateway->disable_subscription_edit_for_india( true, $order );
+
+		// Clean up.
+		\WC_Stripe_Database_Cache::delete( 'mandate_for_subscription_' . $order->get_id() );
+		remove_filter( 'pre_http_request', $mock_request, 10, 3 );
+		WC_Subscriptions_Helpers::$wcs_is_subscription = null;
+
+		$this->assertSame( $expected, $actual );
+	}
```

PHPMD (2.15.0)

[error] The method test_disable_subscription_edit_for_india is not named in camelCase. (undefined)

(CamelCaseMethodName)